src/EditorApp.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="extension-main-container">
+    Drag and drop file to canvas or feed.
+    <div class="images-container">
+      <draggable-file :drag-and-drop-payload="example1" />
+      <draggable-file :drag-and-drop-payload="example2" />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import DraggableFile from './editor/DraggableFile.vue'
+
+const example1 = {
+  thumbUrl: 'https://cache-ssl.celtra.io/api/blobs/b6c7e1ee72ec2f0c9b74769ca63e4d179db2bad0c0ef195158bfafbc31069722/example1.png',
```

Comment:
Use some asset from `files-cdn` instead of blobs, it's not a general purpose storage. Eventually we will be deleting unreferenced blobs.